css/style.css

```diff
@@ -16,4 +15,107 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ .header,
+ .ma-img,
+ .footer,
+ .icon{
+    padding: 0 1.5rem;
+    max-width: auto;
+    margin: 0 auto;
+ }
+ .header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+ }
+ nav{
+    display: flex;
+    width: 100%;
+ }
+.navbar{
+    display: flex;
+    list-style: none;
+    font-size: normal;
+    color: lightgrey;
+    padding-left: 8rem;
+   
+}
+.nav-li-a{
+    margin: 20px;
+    text-decoration: none;
+    color: gray;
+}
+
+.karma-logo{
+    max-height: 40px;
+    width: auto;
+    padding-right: 10rem;
+}
+.meet{
+    font-weight: bold;
+}
+.Store{
```

Comment:
This is a small niggle, but it's worth thinking about. You're using a captial letter in this classname, and you've done it nowhere else. This can be annoying as CSS is case senstive so easy to make a mistake. It's up to you though what you want to do